espanol/canudos.md

```diff
@@ -146,3 +146,5 @@ Spiderman al no entender que era lo que [pasaba] (realidad/realidad.md)
 
 solo [Escapó](escapo/escapo.md)
 
+Y llego [Daynerys Targaryen](thrones/daynerys)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.